### DISCUSSION TITLE
[Feature] Image description overlay

### DISCUSSION THREAD

**Discussion by agdamsbo** [damongolding/immich-kiosk#157] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi. Would it be possible to add the option to overlay the image description text for extra context? Thanks for a great project!

**Comment by damongolding** [comment]
Happy to add this as an option when I'm back from vacation 😊

**Reply by agdamsbo** [reply]
Cool! Enjoy! 😊

**Comment by damongolding** [comment]
added in 0.13.0
